### PR TITLE
347 Add e2e Test for tComponent Upload To Local OCI Registry

### PR DESCRIPTION
**Pull request by shivenduverma-sap**

### CODE REVIEW

File: e2e/suite_test.go [hunk 1]
```diff
@@ -40,9 +45,25 @@ func TestMain(m *testing.M) {
 		shared.StartGitServer(namespace),
 		shared.InstallFlux("latest"),
 		shared.RunTiltForControllers("ocm-controller"),
-		shared.ForwardPortForAppName("registry", 5000, stopChannelRegistry),
-		shared.ForwardPortForAppName("gitea", 3000, stopChannelGitea),
+		shared.ForwardPortForAppName("registry", registryPort, stopChannelRegistry),
+		shared.ForwardPortForAppName("gitea", gitRepositoryPort, stopChannelGitea),
 	)
+	// testEnv.AfterEachTest(
```
Comment (Skarlso) [inline e2e/suite_test.go:51]: Don't forget to add these back :)

**Review by phoban01 — APPROVED** [review]
This looks good Shivendu, thanks 👍🏻

**Review by phoban01 — APPROVED** [review]
This looks good Shivendu, thanks 👍🏻

File: e2e/suite_test.go (hunk 1 above)
Comment (shivenduverma-sap) [reply e2e/suite_test.go:51]: This actually wasnt there initially, I was trying to experiment. AfterEachTest() needs to go to individual feature if required

**Pull request merged**